CHANGES.md

```diff
@@ -1,5 +1,73 @@
 # Changelog
 
+## 6.4 (2024-08-01)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/6.3...6.4)
```

Comment:
TODO: Update once #4588 and #4584 are merged.

Reply:
There have been some additional pull requests to fix-up the recent changes to `sct_analyze_lesion`. In total, the needed updates are:

- [ ] #4584
- [ ] #4588
- [ ] #4595
- [ ] #4596

As soon as these PRs are merged, I think we can go ahead with the release. :)